src/components/User/UserDetailComponent.jsx

```diff
@@ -1,26 +1,55 @@
 import React from 'react';
 import { isEmpty } from 'lodash';
 import PropTypes from 'prop-types';
-import { Header, Icon, Table, Segment } from 'semantic-ui-react';
+import { Header, Icon, Table, Segment, Card } from 'semantic-ui-react';
 import LoaderComponent from '../../components/LoaderComponent';
 import TableRowDetail from '../TableRowComponent';
 
 import '../../_css/UserDetailComponent.css';
 
-const fakeAssetTable = [
-  {
-    id: 45,
-    asset_type: 'Computer',
-    asset_code: 'AND/RE/34',
-    serial_number: 'GT2342342'
-  },
-  {
-    id: 56,
-    asset_type: 'Dongle',
-    asset_code: 'AND/DNG/43',
-    serial_number: 'DNG90909903'
+const assetsAssigned = (allocatedAssets) => {
+  if (!isEmpty(allocatedAssets)) {
+    return (
+      <Table basic selectable>
+        <Table.Header>
+          <Table.Row>
+            <Table.HeaderCell>
+              Asset Type
+            </Table.HeaderCell>
+            <Table.HeaderCell>
+              Asset Code
+            </Table.HeaderCell>
+            <Table.HeaderCell>
+              Serial Number
+            </Table.HeaderCell>
+          </Table.Row>
+        </Table.Header>
+        <Table.Body>
+          {
+            allocatedAssets.map(asset => (
+              <TableRowDetail
+                key={asset.id}
+                data={asset}
+                headings={[
+                  'asset_type',
+                  'asset_code',
+                  'serial_number'
+                ]}
+              />
+            ))
+          }
+        </Table.Body>
+      </Table>
+    );
   }
-];
+  return (
```

Comment:
Do the check the other way round. I.e.
```
if (isEmpty(allocatedAssets)) {
  return (
   <Card>...</Card>
  );
}

return (
  <Table basic selectable>...</Table>
);

Reply:
@sachungo Ha ha ha! Part of me did not want to leave.
 Let me work on that comment.